src/utils/constants/packageAccounts.ts

```diff
@@ -0,0 +1,44 @@
+import { env } from "../../config";
+
+type AccountType = Record< "name" | "description" | "fee", string | string |string>;
+
+type AccountTypes = Record<string, AccountType>;
+
+const PackageAccounts: AccountTypes =  {
+    cic: { name: "cic", description: "Community Interest Company (CIC) accounts", fee: env.CIC_FEE },
+    overseas: { name: "overseas", description: "Overseas company accounts", fee: env.OVERSEAS_FEE },
+    "audit-exempt-subsidiary": { name: "audit-exempt-subsidiary", description: "Audit exempt subsidiary accounts", fee: "-" },
+    "filing-exempt-subsidiary": { name: "filing-exempt-subsidiary", description: "Dormant exempt subsidiary accounts", fee: "-" },
+    "limited-partnership": { name: "limited-partnership", description: "Limited partnership accounts", fee: "-" },
+    uksef: { name: "uksef", description: "UKSEF accounts for a listed company", fee: "-" },
+    "group-package-400": { name: "group-package-400", description: "Group package accounts - section 400, parent incorporated under UK law", fee: "-" },
+    "group-package-401": { name: "group-package-401", description: "Group package accounts - section 401, parent incorporated under non UK Law", fee: "-" },
+    welsh: { name: "welsh", description: "Welsh accounts with an English translation", fee: "-" }
+};
+
+function getAccountType(packageType: string): AccountType {
+    return PackageAccounts[packageType];
+}
+
+type PackageAccountType = Record<string, string | Record<string, string | null>>;
+const packageItems: Array<PackageAccountType> = [];
```

Comment:
So these are objects used to populate the radio buttons on screen. Maybe we should give it a name that describes that like packageChoiceRadioButtons or something similar